Components/Image/image.css

```diff
@@ -0,0 +1,18 @@
+.custom-image-container {
+  width: 100%;
+  height: 300px;
```

Comment:
use rem here as well

Reply:
didnt want it to scale but changed units anyway :) thanks